addons/ingress-nginx/resources/daemonset.yml.erb

```diff
@@ -25,6 +25,13 @@ spec:
       <%- end -%>
       serviceAccountName: nginx-ingress-serviceaccount
       hostNetwork: true
+      tolerations:
```

Comment:
Probably should not render `tolerations:` at all if not set?

Reply:
The end result is the same, there's no tolerations. :) But I can guard that too if needed

Reply:
I think it looks better if guarded.

Reply:
done